### PR TITLE
Update UG for parallel languages

### LINKED ISSUE #280: UG: parallel language
No parallel language in the convention section

image

image

### LINKED ISSUE #284: UG: Command Parameters
Change to a list of prefixes and their respective constraints

image

### LINKED ISSUE #288: UG: Table parameters/description standardization
Some are capitalized, and some are not. Update it to be grammatically correct.

Command Format:
image
* Update it to be small letters

Other Misc:
**TAB**
image

**ADD**
image

**EDIT**
image

### LINKED ISSUE #289: UG: Update PRACTICALEXAM to PRACTICAL_EXAM

### PR DESCRIPTION
**Pull request by raydenlim**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -45,16 +45,16 @@ If you are new to CLI, or unfamiliar with the commands F.A.K.E.J.A.R.V.I.S. offe
...
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 5.1.2 [🔀 Switching between different tabs : `tab`](#switching-between-different-tabs-tab)  
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 5.1.1 [👀 Viewing Help : `help`](#viewing-help-help)  
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 5.1.2 [🔀 Switching between different Tabs : `tab`](#switching-between-different-tabs-tab)  
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 5.1.3 [🏃🚪 Exiting F.A.K.E.J.A.R.V.I.S. : `exit`](#exiting-f-a-k-e-j-a-r-v-i-s-exit)  
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 5.1.4 [💾 Saving the Data](#saving-the-data)  
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 5.1.5 [🛠️ Editing the Data File](#editing-the-data-file)  
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 5.1.6 [💾 Archiving Data Files `[coming in v2.0]`](#archiving-data-files-coming-in-v2-0)
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 5.1.6 [💾 Archiving the Data Files `[coming in v2.0]`](#archiving-data-files-coming-in-v2-0)
```
Comment (wesho1107) [inline docs/UserGuide.md]: Not sure if "the" are required here, does not seem logical and no previous reference to what "the" Data File is before this.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -426,7 +443,7 @@ If your changes to the data file makes its format invalid, F.A.K.E.J.A.R.V.I.S.
 
 <br>
 
-#### 💾Archiving Data Files `[coming in v2.0]`
+#### 💾Archiving the Data Files `[coming in v2.0]`
```
Comment (wesho1107) [inline docs/UserGuide.md]: Refer to previous comment

**Review by wesho1107 — COMMENTED** [review]
Other than a few grammatical nits, LGTM!

File: docs/UserGuide.md (hunk 2 above)
Comment (raydenlim) [reply docs/UserGuide.md]: reverted changes :)

**Review by Respirayson — APPROVED** [review]
LGTM, good catches on some of the capitalisations

**Pull request merged**